aten/src/ATen/core/Tensor.h

```diff
@@ -254,6 +254,8 @@ class CAFFE2_API Tensor {
   template<typename T>
   T * data() const;
 
+  uint8_t* int_repr() const;
```

Comment:
@gchanan - is it fine that it returns different thing in c++ and python? in python it can't return just pointer and likely we should just return a tensor